src/arrayMethodSort.js

```diff
@@ -4,8 +4,23 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(
+    compareFunction = (a, b) => String(a) > String(b)
```

Comment:
The compare function does not correctly handle all cases. It should return a negative, zero, or positive value, not a boolean. A more correct default comparison could be (a, b) => String(a).localeCompare(String(b)).

Reply:
https://mate.academy/chats/27190?selectedMessageId=1401459